themes/tiddlywiki/vanilla/base.tid

```diff
@@ -1877,10 +1877,6 @@ a.tc-tiddlylink.tc-plugin-info:hover .tc-plugin-info > .tc-plugin-info-chunk > s
 	padding: 1em 1em 1em 1em;
 }
 
-/*
-** Checkbox tiddlers transcluded into ControlPanel. see: #1954
-*/
-
```

Comment:
The other sections also have comments like those. You may want to remove #1954. but even this one is useful. imo.